csharp/ql/integration-tests/posix-only/standalone_dependencies/Assemblies.expected

```diff
@@ -168,5 +164,4 @@
 | /microsoft.netcore.app.ref/7.0.2/ref/net7.0/WindowsBase.dll |
 | /microsoft.netcore.app.ref/7.0.2/ref/net7.0/mscorlib.dll |
 | /microsoft.netcore.app.ref/7.0.2/ref/net7.0/netstandard.dll |
-| /newtonsoft.json/12.0.1/lib/portable-net45+win8+wp8+wpa81/Newtonsoft.Json.dll |
-| /nunit/3.13.3/lib/netstandard2.0/nunit.framework.dll |
```

Comment:
Is it expected that this `/nunit/3.13.3/lib/netstandard2.0/nunit.framework.dll` was removed? Is it a transitive reference from one of the framework packages?

Reply:
That line should actually be removed, but not for the reason that I got removed here.
In the standalone integration test projects the NUnit package reference is commented out, but the logic in FileContent doesn't check whether the Package references are commented out - so it will be listed as a missing package and then the missing package fetching logic downloads it.
In PR https://github.com/github/codeql/pull/14665 we fix this issue (so it needs to be merged before this PR).

Reply:
Ohh, I see.
We could also check if we can uncomment the package reference. I don't remember any longer why I commented it out.

Reply:
Sure, I will also try that (but not in this PR).

Reply:
It turns out that the comment in the test is still valid: The DB-CHECK fails.